lms/djangoapps/courseware/tests/test_submitting_problems.py

```diff
@@ -1069,22 +1070,6 @@ def test_other_data_types(self):
                     },
                 }
             )
-        for student_module in student_modules:
-            if student_module.module_state_key.name == problem_name:
-                for val in ('Correct', True, False, 0, 0.0, 1, 1.0, None):
-                    state = json.loads(student_module.state)
-                    state["student_answers"]['i4x-MITx-100-problem-p1_2_1'] = val
-                    student_module.state = json.dumps(state)
-                    student_module.save()
-
-                    self.assertEqual(
-                        grades.answer_distributions(self.course.id),
-                        {
-                            ('p1', 'p1', 'i4x-MITx-100-problem-p1_2_1'): {
-                                str(val): 1
-                            },
-                        }
-                    )
```

Comment:
If you expand up a bit, you'll see that this section of test already exists. It appears to have been duplicated by mistake-- possibly as a result of an errorduring conflict resolution. The upper one was written more cleanly, so I kept it.